config/locales/en.yml

```diff
@@ -97,6 +97,7 @@ en:
     asn_full_name: "Autonomous System Number"
     average_score: "Average score for validators from ASN."
     commission: "Commission is the percent of network rewards earned by a validator that are deposited into the validator's vote account."
+    delinquent: "Delinquent validator is a validator that does not give rewards for the epoch for some period of time."
```

Comment:
```suggestion
    delinquent: "Delinquent validator is a validator that has 100% of commission in epoch."
```

Reply:
Do you have some source of this defintion? I don't think it's correct.

Reply:
There's no external source - this is the term that we (validators) introduced.

Reply:
Actually, those are private validator's definition (both mine and yours). You need to check conditions in code for delinquent validato. There two are not correct.

Reply:
We get delinquent field value from CLI request, I can't see any modification made by our code - https://github.com/brianlong/www.validators.app/blob/master/app/logic/solana_logic.rb#L151

Why it's private validator's definitions?

Reply:
`does not give rewards` - this is about private validators, they keep reward. From what I can see, there are many cases in which Solana can mark validator as delinquent, so I don't think it's a good idea to explain it here. Also, I'd use something more informative for the context in which this definition is used, so that the users know what this column represents in this particular case. So instead, something like:
```Numer of delinquent nodes in Data Center```

Reply:
Where did you find this information about marking validator as delinquent by solana? I can't find it anywhere.

Reply:
I couldn't find any direct clear explanation on Solana website either, which is a pity. I switched to Discord some time ago, there's a lot more information there.